src/lib/components/TeamDefaultModel.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { enhance } from '$app/forms'
+  import SmallCards from '$lib/components/SmallCards.svelte'
+  import type { Model } from '$lib/server/api/openai'
+  import Alert from './Alert.svelte'
+
+  export let models: Model[]
+  export let model: string
+  export let form: Record<string, string> = {}
+</script>
+
+<div class="px-4 sm:px-6 lg:px-8 max-w-6xl py-10">
+  <div class="sm:flex-auto">
+    <h1 class="text-base font-semibold leading-6 text-white">Default model</h1>
+    <p class="mt-2 text-sm text-gray-300">
+      The model used per default when a member of your team starts a new chat.
+    </p>
+  </div>
+  <form method="post" action="?/updateTeamModel" use:enhance>
+    <SmallCards
+      loading={false}
+      bind:value={model}
+      options={models.map((x) => ({ value: x.id, label: x.label, enabled: x.enabled }))}
```

Comment:
```suggestion
      options={models.map((model) => ({ value: model.id, label: model.label, enabled: model.enabled }))}
```

Reply:
I fixy fix! I'll make the same changes in `ChatSettingsPopover` too.